tasks/build/Gaia/build.sh

```diff
@@ -23,11 +23,14 @@ for GOOS in $platforms; do
   make get_vendor_deps
   make install
 
-  if [[ "$GOOS" == "linux" ]]; then
+  if [ "$GOOS" = "linux" ]; then
```

Comment:
there is an advantage in switching to the legacy `[` against `[[`? it should be more portable but I think this will anyway run on up to date machines / containers

Reply:
there were issues with the old notation `[[ "$GOOS" == "linux" ]]` on circleci
the whole build process didn't run so we didn't see that this actually would have failed

Reply:
adding: open to trying this again with you

Reply:
That's funny, the `[[` should be the current notation, ok let's stick to the legacy and see in another PR what docker image need to be updated